docs/src/manual/constraints.md

```diff
@@ -169,18 +169,37 @@ julia> @constraint(model, A * x .>= b)
 Inequalities between matrices are not supported, due to the common ambiguity
 between elementwise inequalities and a [`PSDCone`](@ref) constraint.
 
-```jldoctest symmetric_matrix
+````jldoctest symmetric_matrix
```

Comment:
```suggestion
```jldoctest symmetric_matrix
```

Reply:
Nah. This really needs four ticks to disambiguate the inner triple.